### DISCUSSION TITLE
Confusing version number

### DISCUSSION THREAD

**Discussion by Orchys** [peers/peerjs#1224] [Q&A] [answered]
I have followed the usage instructions and all works for me. I am just a little bit confused with version output of peer server. When I run
`$ peerjs --version`
I get
`1.0.0.`
But after starting the server it states
`Started PeerServer on ::, port: 9000, path: / (v. 0.0.0-development)`
Is that some mistake?

**Comment by jonasgloning** [comment, marked as answer]
Thank you for noticing!
Yes, this is a mistake. It will be fixed in the next version.
